elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -60,7 +59,13 @@ async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, adviso
         non_flaw_bugs = [b for b in bugs if b.is_ocp_bug()]
 
         validator.validate(non_flaw_bugs, verify_bug_status, no_verify_blocking_bugs)
-        validator.verify_bugs_advisory_type(non_flaw_bugs, advisory_id_map, advisory_bug_map)
+
+        # skip advisory type check if advisories are
+        # manually passed in and we don't know their type
+        if '?' not in advisory_id_map.keys():
```

Comment:
do this earlier since `categorize_bugs_by_type` used will trip up if it doesn't find advisory type